programs/voter-stake-registry/tests/test_basic.rs

```diff
@@ -72,12 +72,6 @@ async fn test_basic() -> Result<(), TransportError> {
         .create_voter(&registrar, &token_owner_record, &voter_authority, &payer)
         .await;
 
-    // create the voter again, should have no effect
-    context
-        .addin
-        .create_voter(&registrar, &token_owner_record, &voter_authority, &payer)
-        .await;
```

Comment:
hm, why did this test need to be removed?

Reply:
In [this commit](https://github.com/blockworks-foundation/voter-stake-registry/commit/ae7dea95ebe55db19975ecb44d17d09be95ae7d1) _init_if_needed_ was changed to _init_ and test_basic was not updated, so it was failing. Looks like over the lifetime of it has been _init_ -> _init_if_needed_ -> _init_